lua_modules/http/httpserver.lua

```diff
@@ -7,6 +7,14 @@
 local collectgarbage, tonumber, tostring = collectgarbage, tonumber, tostring
 
 local http
+
+-- add some status code
+local status_code = {}
+status_code[200] = "200 OK"
+status_code[304] = "304 Not Modified"
+status_code[404] = "404 Not Found"
+status_code[500] = "500 Internal Server Error"
+
```

Comment:
While I anticipate that `httpserver.lua` will usually end up in LFS, is there any benefit for these?  Does the standard require them, and if so, shouldn't we have the full set or require the user to supply it?  If we keep them, replace "add some status code" with "translate numeric HTTP codes to required descriptive strings" or such.

Reply:
I saw that has a todo( add status code table). so I add this. all the code maybe unnecessary I think 😂

Reply:
Oh I see.  Well, either way's OK by me, I guess. :)